semcore/utils/src/use/useFocusLock.ts

```diff
@@ -219,6 +206,30 @@ const useFocusLockHook = (
     };
   }, [disabled, autoFocus, returnFocusTo, returnFocus]);
 
+  React.useEffect(() => {
```

Comment:
Why did you move the effect?

Reply:
because I need to compare values in focusMastersStack in `returnFocus` before they removed from stack